MediaBrowser.Model/Session/GeneralCommand.cs

```diff
@@ -14,14 +14,16 @@ public GeneralCommand()
     }
 
     [JsonConstructor]
-    public GeneralCommand(Dictionary<string, string> arguments)
+    public GeneralCommand(Dictionary<string, string>? arguments = null)
```

Comment:
Is the default value necessary?

```suggestion
    public GeneralCommand(Dictionary<string, string>? arguments)
```

Reply:
That’s a good question I’ll rebuild it and see

Reply:
Yes this worked